lib/init/grass.py

```diff
@@ -371,6 +371,8 @@ def get_grass_config_dir():
     Configuration directory is for example used for grass env file
     (the one which caries mapset settings from session to session).
     """
+    if 'GRASS_CONFIG_DIR' in os.environ:
```

Comment:
The changes in this file are probably not Cmake related and could be separated out?